custom_components/zha_toolkit/utils.py

```diff
@@ -34,6 +34,9 @@
     # pylint: disable=ungrouped-imports
     from homeassistant.helpers.json import save_json
 
+if parse_version(HA_VERSION) >= parse_version("2024.6"):
+    from homeassistant.helpers import device_registry, entity_registry
```

Comment:
Suggestion based on pattern in Home-Assistant Core: rename the imported registries

```suggestion
    from homeassistant.helpers import device_registry as dr, entity_registry as er
```